tests/D2L.CodeStyle.Analyzers.Test/UnsafeStatics/UnsafeStaticsAnalyzerTests.cs

```diff
@@ -444,11 +497,16 @@ internal class Foo {
                 public string ClientsName = ""YOLO"";
             }
 
-            public static Foo bad { get; }
+            public static Foo bad { get; } = new Foo()

```

Comment:
This was another case where, once the reasons are exposed, the actual test is slightly different than expected. While the test is not wrong, I changed is to that it matches the same reason as its field equivalent (`DocumentWithStaticProperty_ReadonlyButMutable_Diag`).